main.go

```diff
@@ -461,11 +462,27 @@ func main() {
 	router.Run(*listen_addr)
 }
 
+func populateConfigWithEnvironmentVariables(cfg *Config) {
+	telegramToken, found := os.LookupEnv("TELEGRAM_TOKEN")
+	if found {
+		cfg.TelegramToken = telegramToken
+	}
+
+	sendOnly, found := os.LookupEnv("SEND_ONLY")
+	if found {
+		sendOnlyBool, err := strconv.ParseBool(sendOnly)
+		if err != nil {
+			log.Fatalf("Error parsing sendOnly variable: %v", err)
+		}
+		cfg.SendOnly = sendOnlyBool
+	}
+}
+
```

Comment:
This wasn't mentioned in the description and I'm not comfortable accepting it as is. `SEND_ONLY` doesn't seem to be a good env variable name, also that functionality must be covered in README.md.